### DISCUSSION TITLE
Open cache (`--open-cache` option) invalidation

### DISCUSSION THREAD

**Discussion by solracsf** [juicedata/juicefs#3290] [Q&A] [answered]
I have a mount where files properties (metadata and `ctime`, essentially) are read frequently.
To speed up things, i'm evaluating `--open-cache` option.

Since **this is a single mount/client**, can i enable `--open-cache` with a high number (like 7200s) safely?
Is there any **cache invalidation** if cached file by the `--open-cache` option changes somehow (metadata, ctime)?

**Comment by SandyXSD** [comment, marked as answer]
Yes, it will invalidate the open-cache if a file is modified.

**Reply by solracsf** [reply]
Completed by https://github.com/juicedata/juicefs/pull/3335
